template/Makefile.in

```
@@ -29,7 +29,7 @@ CPP = @CPP@
 LD = @LD@
 RUSTC = @RUSTC@
 CARGO = @CARGO@
-YACC = bison
+YACC = $(tooldir)/lrama/exe/lrama
```

`/usr/bin/env` shebang trick is not portable.

```suggestion
YACC = $(BASERUBY) $(tooldir)/lrama/exe/lrama
```